DirectProgramming/DPC++/ParallelPatterns/openmp_reduction/src/main.cpp

```diff
@@ -1,106 +0,0 @@
-//==============================================================
-// Copyright © 2020 Intel Corporation
-//
-// SPDX-License-Identifier: MIT
-// =============================================================
-#include <iomanip>  // setprecision library
-#include <iostream>
-#include "dpc_common.hpp"
-// Many oneAPI code samples share common include files. These
```

Comment:
Please move your comment regarding dpc_common.hpp it should look like this:

// dpc_common.hpp can be found in the dev-utilities include folder.
// e.g., $ONEAPI_ROOT/dev-utilities/<version>/include/dpc_common.hpp
#include "dpc_common.hpp"

as seen in https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/mandelbrot/src/main.cpp